Bindings/Java/Matlab/Dynamic_Walker_Example/CreateWalkingModelAndEnvironment.m

```diff
@@ -86,66 +87,54 @@
-    orientationInParent, platform, locationInChild, orientationInChild, false);
+platformToGround    = WeldJoint('PlatformToGround', ...
+    ground, locationInParent, orientationInParent, ...
+    platform, locationInChild, orientationInChild);
 
-% Add Visible Object for GUI
-platform.addDisplayGeometry('box.vtp');
-platform.updDisplayer().setScaleFactors([PlatformLength, PlatformWidth, 1]);
-platform.updDisplayer().translate(Vec3(PlatformLength/2-PlatformOffset,0,0));
+% Add geometry to display in the GUI
+platformGeomOffset = PhysicalOffsetFrame(platform, ...
+    Transform(Vec3(PlatformLength/2 - PlatformOffset, 0, 0)));
+platformGeomOffset.attachGeometry(...
+    Brick(Vec3(PlatformLength, PlatformWidth, 1)));
+platform.addComponent(platformGeomOffset);
 
 % Add Body to Model
 osimModel.addBody(platform);
+osimModel.addJoint(platformToGround);
 % -----------------------------------------------------------------------
-% Create a massless body for one DOF of the Pelvis
-pelvisMassless      = Body();
-pelvisMassless.setName('Pelvis_massless');
-pelvisMassless.setMass(0);
-pelvisMassless.setInertia(Inertia(0,0,0,0,0,0));
-
-% Create Slider Joint
-locationInParent    = Vec3(0,PlatformWidth/2,0);
-orientationInParent = Vec3(0,0,0);
-locationInChild     = Vec3(0,0,0);
-orientationInChild  = Vec3(0,0,0);
-pelvisMasslessToPlatform    = SliderJoint('PelvisMasslessToPlatform', platform, ...
-    locationInParent, orientationInParent, pelvisMassless, locationInChild, orientationInChild, false);
-
-% Update the coordinates of the new joint
-jointCoordinateSet  = pelvisMasslessToPlatform.upd_CoordinateSet();
-jointCoordinateSet.get(0).setRange([0, 100]);
-jointCoordinateSet.get(0).setName('Pelvis_tx');
-jointCoordinateSet.get(0).setDefaultValue(0);
-
-% Add Body to Model
-osimModel.addBody(pelvisMassless);
-% -----------------------------------------------------------------------
-% Create a massive body for the last DOF of the pelvis
+% Create a massive body for the 6 DOF of the pelvis
 pelvis              = Body();
 pelvis.setName('Pelvis');
 pelvis.setMass(PelvisMass);
 pelvis.setInertia(Inertia(smallInertia,PelvisIYY,smallInertia,0,0,0));
 
-% Create a Slider Joint
-locationInParent    = Vec3(0,0,0);
-orientationInParent = Vec3(0,0,pi/2);
-locationInChild     = Vec3(0,0,0);
-orientationInChild  = Vec3(0,0,pi/2);
-pelvisToPelvisMassless    = SliderJoint('PelvisToPelvisMassless', pelvisMassless, ...
-    locationInParent, orientationInParent, pelvis, locationInChild, orientationInChild, false);
+% Create Planar Joint
+pelvisToPlatform    = PlanarJoint('PelvisToPlatform', platform, pelvis);
 
 % Update the coordinates of the new joint
-jointCoordinateSet  = pelvisToPelvisMassless.upd_CoordinateSet();
-jointCoordinateSet.get(0).setRange([0, 5]);
-jointCoordinateSet.get(0).setName('Pelvis_ty');
-jointCoordinateSet.get(0).setDefaultValue(1.0);
+Pelvis_rz = pelvisToPlatform.upd_coordinates(0);
```

Comment:
It seems a bit inconsistent to start using variables with capital first letter.

Reply:
I don't like the use of capital first letters for variables, but I did it to be consistent with the rest of the file.

Reply:
That's fine. I'm happy with the PR.